tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -2723,6 +2723,26 @@ def take_while(self, predicate):
 
     return _TakeWhileDataset(self, predicate)
 
+  def unique(self):
+    """A transformation that discards duplicate elements of a `Dataset`.
+
+    Use this transformation to produce a dataset that contains one instance of
+    each unique element in the input. For example:
+
+    >>> dataset = tf.data.Dataset.from_tensor_slices([1, 37, 2, 37, 2, 1])
+    >>> dataset = dataset.unique()
+    >>> sorted(list(dataset.as_numpy_iterator()))
+    [1, 2, 37]
+
+    NOTE: This transformation only supports datasets which fit into memory
```

Comment:
replace "NOTE" with "Note" because the former is not treated by markdown visualization as a callout

Reply:
addressed in the commit.